src/coremods/core_dns.cpp

```diff
@@ -344,7 +344,7 @@ class Packet : public Query
 				}
 				else
 				{
-					unsigned long forward = ip.in4.sin_addr.s_addr;
+					in_addr_t forward = ip.in4.sin_addr.s_addr;
```

Comment:
Does this typedef exist on Windows? If not you might need to add a typedef to `inspircd_win32wrapper.h` for it.

Reply:
it should map to `IN_ADDR` i think, but then i'm surprised that access to `s_addr`  works on windows because i _think_ it [should be spelled](https://docs.microsoft.com/en-us/windows/win32/api/winsock2/ns-winsock2-in_addr#:~:text=The%20in_addr%20structure%20is%20the,with%20Windows%20Vista%20and%20later.&text=On%20earlier%20versions%20of%20the,be%20declared%20as%20struct%20in_addr.) `S_addr`.

i put this back to `unsigned int` :)